clients/naming_client/naming_grpc/connection_event_listener.go

```diff
@@ -98,9 +98,8 @@ func (c *ConnectionEventListener) CacheInstanceForRedo(serviceName, groupName st
 	getInstance, ok := c.registeredInstanceCached.Get(key)
 	if !ok {
 		logger.Warnf("CacheInstanceForRedo get cache instance is null,key:%s", key)
```

Comment:
instance is no longer in the cache . no need to print the log. can you delete it